### PR TITLE
Image resolution

### PR DESCRIPTION
**Pull request by typemytype**
see #215 

todo:

* write docs
* add tests

### CODE REVIEW

File: drawBot/drawBotDrawingTools.py [hunk 1]
```diff
@@ -1978,6 +1978,33 @@ def imagePixelColor(self, path, xy):
...
+    def imageResolution(self, path):
+        """
+        Return the image resolution for a given image.
+        """
+        if isinstance(path, AppKit.NSImage):
+            # its an NSImage
+            # get the bitmap representation
+            try:
+                rep = path.representations()[0]
+            except Exception:
```
Comment (justvanrossum) [inline drawBot/drawBotDrawingTools.py]: Isn't it a simple IndexError you want to catch here?

File: drawBot/drawBotDrawingTools.py [hunk 2]
```diff
@@ -1978,6 +1978,33 @@ def imagePixelColor(self, path, xy):
...
+                path = optimizePath(path)
+            if path.startswith("http"):
+                url = AppKit.NSURL.URLWithString_(path)
+            else:
+                if not os.path.exists(path):
+                    raise DrawBotError("Image does not exist")
+                url = AppKit.NSURL.fileURLWithPath_(path)
+                try:
+                    rep = AppKit.NSImageRep.imageRepWithContentsOfURL_(url)
+                except Exception:
```
Comment (justvanrossum) [inline drawBot/drawBotDrawingTools.py]: Can this exception be more specific, too?

Comment (typemytype) [reply drawBot/drawBotDrawingTools.py]: this catches a range of errors:  file not found, file not readable, file not valid, ...

File: drawBot/drawBotDrawingTools.py (hunk 1 above)
Comment (typemytype) [reply drawBot/drawBotDrawingTools.py]: fixed

**Pull request merged**